yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Seems like you ran yarn on the root instead of inside of the `website` folder. Can you remove this file again before merging?